tests/wpt/metadata/dom/nodes/ParentNode-querySelector-All.html.ini

```diff
@@ -144,30 +90,12 @@
   [Fragment.querySelector: ::first-letter pseudo-element (two-colon syntax) selector, not matching any elements: #pseudo-element::first-letter]
     expected: FAIL
 
-  [In-document Element.querySelectorAll: :lang pseudo-class selector, matching inherited language: #pseudo-lang-div1:lang(en)]
-    expected: FAIL
-
-  [In-document Element.querySelector: :lang pseudo-class selector, matching inherited language: #pseudo-lang-div1:lang(en)]
-    expected: FAIL
-
-  [In-document Element.querySelectorAll: :lang pseudo-class selector, matching specified language with exact value: #pseudo-lang-div2:lang(fr)]
-    expected: FAIL
-
-  [In-document Element.querySelector: :lang pseudo-class selector, matching specified language with exact value: #pseudo-lang-div2:lang(fr)]
-    expected: FAIL
-
   [In-document Element.querySelectorAll: :lang pseudo-class selector, matching specified language with partial value: #pseudo-lang-div3:lang(en)]
     expected: FAIL
 
   [In-document Element.querySelector: :lang pseudo-class selector, matching specified language with partial value: #pseudo-lang-div3:lang(en)]
```

Comment:
Similarly. I guess this is about the extended filtering `FIXME`? If so, please file an issue.